.travis.yml

```diff
@@ -1,6 +1,10 @@
 language: python
 python:
-  - "2.7"
+  - "3.7"
```

Comment:
That's a tricky thing: if we want to merge it upstream, we shouldn't break Python 2 support, so we should also launch the tests against it.

Reply:
That's true, although it's going to be a tough task. For instance, how do people usually tackle things like syntax change in `except` clause?

Reply:
And support for Python 2.7 ends in a couple of months anyway, so...

Reply:
There's a lot of libraries out there which support both major Python versions. I guess it's a case of what the original author had in mind. We can drop Python 2 support now & when we submit a PR to the upstream & the author complains about that, we'll work on re-adding that.